### ISSUE TITLE
texlive: sha256 has changed

### ISSUE THREAD

**Issue by rwhogg** [Linuxbrew/homebrew-core#2318]
Same deal as #1425 

```
Error: SHA256 mismatch
Expected: 61905f2a3423ce0c66837063968ff336fd82b9a0279dd20917beecee6fd3f35f
Actual: 645897c1001c75d85da147dc2dc6f7dc6d9ddd404f3dfc83050242232c658c1a
Archive: /home/bob/.cache/Homebrew/texlive-20170111.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by sjackman** [comment]
To get the new version number and SHA-256
```sh
curl -L http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz | tar t | head -n1
curl -L http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz | sha256sum
```

**Comment by sjackman** [comment]
Fixed by #2319
